pandas/io/formats/style_render.py

```diff
@@ -789,6 +792,38 @@ def _translate_latex(self, d: dict) -> None:
             body.append(row_body_headers + row_body_cells)
         d["body"] = body
 
+        # clines are determined from info on index_lengths and hidden_rows and input
```

Comment:
woudn't object to putting this into a separate function and calling here

Reply:
took a look but the logic depends on a few vars created in the 10 lines above and a separate func would need to be passed a few vars, inc `self` which gets messier. think is better just to leave here as self contained block.